spec/factories/state_file_id_intakes.rb

```diff
@@ -192,9 +192,11 @@
 
     trait :with_eligible_1099r_income do
       after(:create) do |intake|
-        create(:state_file1099_r, intake: intake, taxable_amount: 2000, state_tax_withheld_amount: 200) do |form_1099r|
+        create(:state_file1099_r, intake: intake, taxable_amount: 2000, state_tax_withheld_amount: 200, recipient_ssn: intake.primary.ssn) do |form_1099r|
           create(:state_file_id1099_r_followup, state_file1099_r: form_1099r, eligible_income_source: "yes")
         end
+
+        intake.update(primary_disabled: "yes")
```

Comment:
there's a spec of the id39r calculator that utilizes these traits `:with_eligible_1099r_income` and `:without_eligible_1099r_income` that demonstrates the change